modules/nw-sriov-nic-mlx-secure-boot.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-nic-mlx-secure-boot_{context}"]
+= Configuring the SR-IOV Network Operator on Mellanox cards when Secure Boot is enabled
+
+The SR-IOV Network Operator supports an option to skip the firmware configuration for Mellanox (MLX) devices. It is currently the only way to create virtual functions by using the SR-IOV Network Operator, if the system has secure boot enabled. You are required to manually configure and allocate the number of virtual functions in the firmware before switching the system to secure boot.
+
+[NOTE]
+====
+The number of virtual functions in the firmware are the maximum number of virtual functions that you can request in the policy.
+====
+
+.Procedure
+
+. Configure the virtual functions (VFs):
+
+.. Run the following command when the system is without a secure boot when using the sriov-config daemon:
++
+[source,terminal]
+----
+$ mstconfig -d -0001:b1:00.1 set SRIOV_EN=1 NUM_OF_VFS=16 <1> <2>
+----
+<1> The `SRIOV_EN` environment variable enables the SR-IOV Network Operator support on the Mellanox card.
+<2> The `NUM_OF_VFS` environment variable specifies the number of virtual functions to enable in the firmware.
+
+. Configure the SR-IOV Network Operator by disabling the Mellanox plugin. See the following `SriovOperatorConfig` example configuration:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  configDaemonNodeSelector: {}
+  configurationMode: daemon
+  disableDrain: false
+  disablePlugins:
+  - mellanox
+  enableInjector: true
+  enableOperatorWebhook: true
+  logLevel: 2
+----
+
+. Reboot the system to enable the virtual functions and the configuration settings.
+
+. Check the virtual functions (VFs) after rebooting the system by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-sriov-network-operator get sriovnetworknodestate.sriovnetwork.openshift.io worker-0 -oyaml
+----
++
+[source,yaml]
```

Comment:
Is this sample output?
```suggestion
.Example output
[source,yaml]
```

Reply:
Thanks @jeana-redhat for your feedback. Yes. I have applied your suggestion.